README.md

```diff
@@ -51,11 +51,12 @@ python petric.py
 <!-- br -->
 
 > [!WARNING]
-> To avoid timing out (5 min runtime), please disable any debugging/plotting code before submitting!
+> To avoid timing out (currently 10 min runtime for intermediate leaderboard, and 1 hour for final evaluation after submissions close), please disable any debugging/plotting code before submitting!
```

Comment:
I'd certainly hope that we don't need 1 hour for the final evaluation, so I wouldn't mention this.  In any case, our leaderboard needs to be sensible such that people can see some trends. I'm not sure if 10mins will be enough for that, but we can always raise it later.

Reply:
```suggestion
> To avoid timing out (currently 10 min runtime for intermediate leaderboard, which will be adjusted closer to the deadline), please disable any debugging/plotting code before submitting!
```

Reply:
your slides say 1 hour :sweat_smile:

Reply:
wanted to be safe I guess. Anyway, fine for me. just merge then!

Reply:
ok reworded & merged :)